rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -328,13 +336,37 @@ def test_price_underlying_tokens(inquirer, globaldb):
     assert price == FVal(67)
 
 
+@pytest.mark.vcr()
 @pytest.mark.parametrize('use_clean_caching_directory', [True])
-@pytest.mark.parametrize('should_mock_current_price_queries', [True])
-def test_find_uniswap_v2_lp_token_price(inquirer, ethereum_manager):
-    address = '0xa2107FA5B38d9bbd2C461D6EDf11B11A50F6b974'
-    identifier = ethaddress_to_identifier(address)
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_find_uniswap_v2_lp_token_price(inquirer, ethereum_manager, globaldb):
+    """Tests that the Uniswap v2 lp token's price is correctly found. The special price
+    calculation that is needed, is applied based on the protocol attribute of the lp token"""
+    identifier = ethaddress_to_identifier(string_to_evm_address('0xa2107FA5B38d9bbd2C461D6EDf11B11A50F6b974'))  # LINK ETH POOL  # noqa: E501
+    with globaldb.conn.write_ctx() as write_cursor:
+        write_cursor.execute(  # the protocol attribute is missing from the packaged db for this token and is necessary for price calculation  # noqa: E501
```

Comment:
Two things here.

1. This comment should mention the time
2. Making this small fix for the packaged global DB would not be bad if the token is already there and you just fix the protocol. (but will merge now as it's already green -- can adjust later)
```suggestion
        write_cursor.execute(  # the protocol attribute is missing from the packaged db as of this commit for this token and is necessary for price calculation  # noqa: E501
```